### PR TITLE
Master wowl aab

### PR DESCRIPTION
**Pull request by aab-odoo**

### CODE REVIEW

File: addons/wowl/static/src/services/http_service.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/** @odoo-module **/
+
+import { serviceRegistry } from "../webclient/service_registry";
+
+export const httpService = {
+  name: "http",
```
Comment (Polymorphe57) [inline addons/wowl/static/src/services/http_service.js]: remove name

File: addons/wowl/static/tests/components/file_input_tests.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+const { mount } = owl;
+
+// -----------------------------------------------------------------------------
+// Helpers
+// -----------------------------------------------------------------------------
+
+async function createFileInput(config) {
+  const serviceRegistry = new Registry();
+  const fakeHTTPService = {
+    name: "http",
```
Comment (Polymorphe57) [inline addons/wowl/static/tests/components/file_input_tests.js]: same

File: addons/wowl/static/tests/components/file_input_tests.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    deploy() {
+      return {
+        post: config.mockPost || (() => {}),
+      };
+    },
+  };
+  serviceRegistry.add(fakeHTTPService.name, fakeHTTPService);
```
Comment (ged-odoo) [inline addons/wowl/static/tests/components/file_input_tests.js]: => name

File: addons/wowl/static/src/services/http_service.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  deploy() {
+    return {
+      async get(route, readMethod = "json") {
+        const response = await fetch(route, { method: "GET" });
+        response[readMethod]();
+      },
+      async post(route, params = {}, readMethod = "json") {
+        const formData = new FormData();
+        for (const key in params) {
+          if (key === "method") {
```
Comment (ged-odoo) [inline addons/wowl/static/src/services/http_service.js]: remove the if

**Pull request merged**